alpha/apps/kaltura/modules/kmcng/actions/kmcngAction.class.php

```diff
@@ -78,10 +78,14 @@ private function initConfig($deployUrl, $kmcngParams, $enforceSecureProtocol, $r
 		$this->contentUiconfsLivea = isset($this->liveAUiConf) ? array_values($this->liveAUiConf) : null;
 		$this->contentUiconfLivea = (is_array($this->contentUiconfsLivea) && reset($this->contentUiconfsLivea)) ? reset($this->contentUiconfsLivea) : null;
 
-		$this->previewUIConf = uiConfPeer::getUiconfByTagAndVersion('KMCng', $kmcngParams["kmcng_version"]);
+		$this->previewUIConf = uiConfPeer::getUiconfByTagAndVersion('KMCngV2', $kmcngParams["kmcng_version"]);
 		$this->contentUiconfsPreview = isset($this->previewUIConf) ? array_values($this->previewUIConf) : null;
 		$this->contentUiconfPreview = (is_array($this->contentUiconfsPreview) && reset($this->contentUiconfsPreview)) ? reset($this->contentUiconfsPreview) : null;
 
+		$this->previewUIConfV7 = uiConfPeer::getUiconfByTagAndVersion('KMCngV7', $kmcngParams["kmcng_version"]);
+		$this->contentUiconfsPreviewV7 = isset($this->previewUIConfV7) ? array_values($this->previewUIConfV7) : null;
+		$this->contentUiconfPreviewV7 = (is_array($this->contentUiconfsPreviewV7) && reset($this->contentUiconfsPreviewV7)) ? reset($this->contentUiconfsPreviewV7) : null;
```

Comment:
Useless to check reset($this->contentUiconfsPreviewV7), if its array - take reset, it will contain false